Invalid @Generated date field javadoc comments break javadoc generation.

Hi @hauner!

Plain ```'&'``` character in comment over date field of @Generated annotation breaks javadoc processing.

The output of javadoc run looks like this:
```
.../api/generated/support/Generated.java:25: error: bad HTML entity
     * The date & time of generation (ISO 8601) or "-" if no date was set.
                ^
```

Processed javadoc comments should be a valid HTML.

One solution might be to replace ```'&'``` character with ```'&amp;'```.

--- a/src/main/kotlin/io/openapiprocessor/core/writer/java/GeneratedWriterImpl.kt
+++ b/src/main/kotlin/io/openapiprocessor/core/writer/java/GeneratedWriterImpl.kt
@@ -46,7 +46,7 @@ class GeneratedWriterImpl(private val info: GeneratedInfo, private val options: 
                 String version();
 
                 /**
-                 * The date & time of generation (ISO 8601) or "-" if no date was set.
+                 * The date &amp; time of generation (ISO 8601) or "-" if no date was set.
                  *                 
                  * @return date of generation
                  */

